ts/model_server.py

```diff
@@ -36,8 +36,6 @@ def start():
         else:
             try:
                 parent = psutil.Process(pid)
-                for child in parent.children(recursive=True):
```

Comment:
@maaquib Have you verified that removing the code for terminating child processes does not have any other side effects?

Reply:
@harshbafna Wouldn't this leave orphaned child process? Why are we removing this?

Reply:
@maaquib  : The child processes are nothing else but the Java threads . Now this is being taken care by the shutdown hook added for stopping all the threads and generating the snapshot. Thus this piece of code is not required anymore. 

We have also verified that there are no orphaned java/python processes left running after shutdown. Even tested with benchmark.